src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -949,16 +948,16 @@ public bool TryGetCustomAttributeName(CustomAttributeHandle customAttribute, Met
                     return false;
                 EntityHandle? container = ctorHandle.Kind switch
                 {
-                    HandleKind.MethodDefinition => metadataReader.GetMethodDefinition((MethodDefinitionHandle)ctorHandle).GetDeclaringType(),
-                    HandleKind.MemberReference => metadataReader.GetMemberReference((MemberReferenceHandle)ctorHandle).Parent,
+                    HandleKind.MethodDefinition => EnCGetMethodDefinition((MethodDefinitionHandle)ctorHandle).GetDeclaringType(),
+                    HandleKind.MemberReference => EnCGetMemberReference((MemberReferenceHandle)ctorHandle).Parent,
                     _ => null,
                 };
                 if (container == null)
                     return false;
                 StringHandle? attributeTypeNameHandle = container.Value.Kind switch
                 {
-                    HandleKind.TypeDefinition => metadataReader.GetTypeDefinition((TypeDefinitionHandle)container.Value).Name,
-                    HandleKind.TypeReference => metadataReader.GetTypeReference((TypeReferenceHandle)container.Value).Name,
+                    HandleKind.TypeDefinition => EnCGetTypeDefinition((TypeDefinitionHandle)container.Value).Name,
+                    HandleKind.TypeReference => EnCGetTypeReference((TypeReferenceHandle)container.Value).Name,
```

Comment:
- Do IIUC that these new methods would work irrespective of whether EnC was used with the assembly or not?
- Should we check the other places where we currently use `*.GetTypeDefinition` etc, to see if they need to use these new methods too?

Reply:
1) Yes
2) I checked and I think we should use only in this place.